core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -576,19 +576,13 @@ class KafkaConfig private(doLog: Boolean, val props: util.Map[_, _])
       throw new ConfigException(s"Disabling the '${GroupType.CLASSIC}' protocol is not supported.")
     }
     if (protocols.contains(GroupType.CONSUMER)) {
-      if (processRoles.isEmpty) {
```

Comment:
I had to soften the validation here because ZK also has the new default value now. Keep in mind that ZK will be remove entirely in 4.0 so it won't be an issue for the next release.